src/components/Map.js

```diff
@@ -41,48 +41,37 @@ type Props = any;
 
 export default class SimpleExample extends Component<Props, State> {
   state = {
-    lng: 25.066105,
+    lng: 24.93,
     lat: 60.18808,
-    zoom: 14
+    zoom: 11.47
```

Comment:
magic

Reply:
It is based on carefully analyzing human eye preference and mathematically calculating visually best outcome.